Telerik.Examples.Mvc/Telerik.Examples.Mvc/Hubs/GridHub.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Telerik.Examples.Mvc.Models;
+using Microsoft.AspNetCore.SignalR;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+
+namespace Telerik.Examples.Mvc.Hubs
+{
+    public class GridHub : Hub
+    {
+
+        public override System.Threading.Tasks.Task OnConnectedAsync()
+        {
+            Groups.AddToGroupAsync(Context.ConnectionId, GetGroupName());
+            return base.OnConnectedAsync();
+        }
+
+        public override System.Threading.Tasks.Task OnDisconnectedAsync(Exception e)
+        {
+            Groups.RemoveFromGroupAsync(Context.ConnectionId, GetGroupName());
+            return base.OnDisconnectedAsync(e);
+        }
+
+        public IEnumerable<ProductViewModel> Read()
+        {
+
+            var result = Enumerable.Range(1, 50).Select(i => new ProductViewModel
+            {
+                ProductID = i,
```

Comment:
I would recommend populating the majority of the model's fields. Or, by narrowing the model, as the Update operations appears to falter.